core/domain/exp_domain.py

```diff
@@ -617,6 +619,9 @@ def __init__(
         self.last_updated = last_updated
         self.auto_tts_enabled = auto_tts_enabled
         self.correctness_feedback_enabled = correctness_feedback_enabled
+        self.android_proto_size_in_bytes = None
+        self.proto_size_is_stale = True
```

Comment:
Shouldn't this be android_proto_size_is_stale?

Reply:
Fixed